extensions/downloads-manager/src/open-last-command.tsx

```diff
@@ -0,0 +1,14 @@
+import { open, showHUD } from "@raycast/api";
+import { getDownloads } from "./utils";
+
+export default async function main() {
+  const downloads = getDownloads();
+
+  if (downloads.length < 1) {
+    await showHUD("❌ Failed getting latest download");
```

Comment:
this isn't always an error, right? It's possible that the downloads folder is just empty 🤔